app/controllers/admin/events_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+module Admin
+  class EventsController < ApplicationController
```

Comment:
```suggestion
  class EventsController < ApplicationController
    before_action :authenticate_user!
```

let's add this before action! currently if you're not signed in, hitting this route raises an error since current user is nil